stable/anchore-engine/templates/enterprise_feeds_configmap_env.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.anchoreEnterpriseGlobal.enabled (or .Values.anchoreEnterpriseRbac.enabled .Values.anchoreEnterpriseReports.enabled) -}}
```

Comment:
this should be `{{- if and .Values.anchoreEnterpriseGlobal.enabled .Values.anchoreEnterpriseFeeds.enabled }}`

Reply:
I kept the end of line whitespace chomp `... -}}`.